model/prelude_mem_addrtype.sail

```diff
@@ -0,0 +1,14 @@
+/*=======================================================================================*/
+/*  This Sail RISC-V architecture model, comprising all files and                        */
+/*  directories except where otherwise noted is subject the BSD                          */
+/*  two-clause license in the LICENSE file.                                              */
+/*                                                                                       */
+/*  SPDX-License-Identifier: BSD-2-Clause                                                */
+/*=======================================================================================*/
+
+newtype physaddr = physaddr : xlenbits
```

Comment:
Does not necessarily need fixing for this PR, but the privileged spec defines physical addresses as 34 bits for RV32.

Reply:
Yeah I think there's a long-standing TODO for that. Hopefully this PR will make fixing it a little easier!